app/services/likes/destroyer_service.rb

```diff
@@ -0,0 +1,18 @@
+module Likes
+  class DestroyerService
+    def initialize(like:, photo:, user:)
+      @like = like
+      @photo = photo
+      @user = user
+    end
+
+    def destroy
+      return false if @like.nil?
+
+      @like.liked = false
+      @like.save
+
+      Redis.new.set("#{@photo.id}---#{@user.id}", false)
+    end
```

Comment:
use ruby accessors `attr_reader / writer / accessor` insted of instance variables